### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Simpleshaikh1**
I have completed the task for this session which is to build the desktop version, I added a series of features like
1. grid and flex.
2. row-reverse with flex. 
Maintaining a min-width of 768px.

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @Simpleshaikh1

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **doesn;t have a descriptive PR.** Kindly add a descriptive PR description that contains a summary of all the changes you implemented on this PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/index.css [hunk 1]
```diff
@@ -472,28 +480,186 @@ form #type-msg {
...
+
+  .text-inner-container {
+    width: 90%;
+    margin-right: 5%;
+  }
+
+  .canopy-inner-container ul {
+    width: 60%;
+    margin-left: 5%;
+  }
```
Comment (LionRouge1) [inline css/index.css:580]: - [ ] Nice work your design looks really good. :clap: :sunglasses: However there is too much space between the info about the work, the position, and the year compare to Figma's Design and also the circles that separate them have a grey color (see image). Kindly check it to make your Desktop versin looks like the one in the Figma design.

### Your Design
Screenshot from 2023-04-06 11-50-39

### What it's supposed to look like
Screenshot from 2023-04-06 11-51-38

File: html/index.html [hunk 2]
```diff
@@ -223,30 +228,30 @@ <h2> Tonic </h2>
...
-                    <img src="../assets/Snapshoot Portfolio 4.png" alt="">
+                    <img src="../assets/Snapshoot Portfolio 4.png" alt="" class="mobile-first-card">
+                    <img src="../assets/fourth-desktop-card.png" alt="" class="desktop-first-card">
         
                 </div>
         
-                <div class="text-inner-container">
+                <div class="text-inner-container same">
         
                     <h2> Multi-Post <br> Stories </h2>
```
Comment (LionRouge1) [inline html/index.html:257]: - [ ] Kindly make sure that the projects description text take 3 to 4 lines of text. So you can write more about the project and also make  it more professional.

File: css/index.css [hunk 3]
```diff
@@ -472,28 +480,186 @@ form #type-msg {
...
+
+  .canopy-list {
     display: flex;
     justify-content: space-between;
+    width: 50%;
   }
```
Comment (LionRouge1) [inline css/index.css:587]: - [ ] Great work listing the technologies :+1: :sunglasses: However they same a little bigger and there are more space between them compare to the Figma's Design (see image). Kindly check it to stick as much as possible to the Figma design.

### Your design 
Screenshot from 2023-04-06 12-29-38

### What it's supposed to look like 
Screenshot from 2023-04-06 12-39-11

File: css/index.css [hunk 4]
```diff
@@ -472,28 +480,186 @@ form #type-msg {
...
 
-  .top-body-background {
+  .all-button {
+    margin: 40px 0;
+  }
```
Comment (LionRouge1) [inline css/index.css:590]: - [ ] Good job implementing the buttons :100: However they look a little bigger than the one on the Figma design and also their background color should be white (see image). Kindly check it to show your ability to parse a Figma design to create a UI.

### Your Design
Screenshot from 2023-04-06 11-52-37

### What it supposed to look like
Screenshot from 2023-04-06 11-52-52

File: css/index.css [hunk 5]
```diff
@@ -472,28 +480,186 @@ form #type-msg {
...
+
+  .third-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 0;
+    width: 95%;
+    margin: 0 auto;
+    border-radius: 0 15% 0 0;
+  }
```
Comment (LionRouge1) [inline css/index.css:599]: - [ ] They two blocks in the "About Myself" are nicely aligned :+1: :sunglasses: However in order to follow Figma's design, please make the left one takes 40% and the right one the remaining 60% of the page's width.

### Figma's Design
Screenshot from 2023-04-05 06-38-27

File: css/index.css [hunk 6]
```diff
@@ -472,28 +480,186 @@ form #type-msg {
...
+    text-align: center;
+    width: 70%;
+  }
+
+  .contact-me-section #contact-msg {
+    margin-bottom: 40px;
+    margin-left: 50%;
+    text-align: center;
+    width: 50%;
+  }
```
Comment (LionRouge1) [inline css/index.css:640]: - [ ] Please reduce the height of your contact form message to make it looks like the Figma Design (see image) 

### What it's supposed to look like:
Screenshot from 2023-03-31 15-39-50

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Simpleshaikh1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Your design looks almost identical to Figma design :+1: 
- Your website is responsive :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**